x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_detail.tsx

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useMemo, useEffect, Fragment } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiSpacer,
+  EuiText,
+  EuiDescriptionList,
+  EuiCode,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { CrumbInfo, formatDate, StyledBreadcrumbs } from './panel_content_utilities';
+import * as event from '../../../../common/endpoint/models/event';
+import { ResolverEvent } from '../../../../common/endpoint/types';
+import * as selectors from '../../store/selectors';
+import { useResolverDispatch } from '../use_resolver_dispatch';
+import { PanelContentError } from './panel_content_error';
+
+export const BoldCode = styled(EuiCode)`

```

Comment:
Why is this exported from 'panel_content_related_detail'? maybe remove 'export'?

Reply:
https://github.com/elastic/kibana/pull/69098/commits/7a0c375aaae501f95cbb1d48690d69a1b45ae377